src/rail/estimation/algos/deepdisc.py

```diff
@@ -1,4 +1,31 @@
+import numpy as np
 from rail.estimation.estimator import CatEstimator, CatInformer
+from rail.core.data import TableHandle
+from rail.core.common_params import SHARED_PARAMS
+from detectron2.config import LazyConfig, get_cfg
+import detectron2.solver as solver
+import detectron2.data as data
+from deepdisc.data_format.augment_image import train_augs
+from deepdisc.data_format.image_readers import DC2ImageReader
+from deepdisc.data_format.register_data import register_data_set, register_loaded_data_set
+from deepdisc.model.loaders import (
+    RedshiftFlatDictMapper,
+    RedshiftDictMapper,
+    return_test_loader,
+    return_train_loader,
+)
+from deepdisc.model.models import return_lazy_model
+from deepdisc.training.trainers import (
+    return_evallosshook,
+    return_lazy_trainer,
+    return_optimizer,
+    return_savehook,
+    return_schedulerhook,
+)
+from rail.deepdisc.configs import *
+from deepdisc.inference.match_objects import get_matched_object_classes_new, get_matched_z_pdfs_new, get_matched_z_points_new
```

Comment:
I don't see `get_matched_z_points_new` in the main branch of deepdisc. Is this a function that you have in your local branch?